src/graph/test/TestEnv.cpp

```diff
@@ -45,9 +46,19 @@ void TestEnv::SetUp() {
         LOG(ERROR) << "Bad local host addr, status:" << hostRet.status();
     }
     auto& localhost = hostRet.value();
+
+    std::string clusterIdPath = "/tmp/storage.cluster.id.";
```

Comment:
Just pass the default cluster id to clusterMan is enough here.
We have no needs to read file in UTs

Reply:
Fixed!